lib/LaTeXML/Post/Manifest/Epub.pm

```diff
@@ -198,8 +217,9 @@ sub process {
       $item->setAttribute('href',       $item_url);
       $item->setAttribute('media-type', "application/xhtml+xml");
       my @properties;
-      push @properties, 'mathml' if $doc->findnode('//*[local-name() = "math"]');
-      push @properties, 'svg'    if $doc->findnode('//*[local-name() = "svg"]');
+      push @properties, 'scripted' if $doc->findnode($xpath_scripted . ' | ' . $xpath_forms);
+      push @properties, 'mathml'   if $doc->findnode($xpath_mathml);
+      push @properties, 'svg'      if $doc->findnode($xpath_svg);
```

Comment:
These changes assume we'll only ever have xhtml in epub.  It's at least something to watch out for.

Reply:
This one is easy to explain: the EPUB specs (both 2 and 3 as far as I know) require HTML5 in XML serialisation, with the appropriate namespaces. So this code should be ok, and more accurate, actually.